packages/block-library/src/social-link/edit.js

```diff
@@ -141,6 +146,41 @@ const SocialLinkEdit = ( {
 
 	return (
 		<>
+			{ isContentOnlyMode && showLabels && (
```

Comment:
When using the block normally (not in contentOnly), it's possible to always edit the text, and it's used for screen readers, so I wonder if it's worth considering making it always show in content only mode too. 🤔

Reply:
Just came here to ask this too. It seems like a good follow up to allow editing both text and links. 

Here's "Design" mode when I click on a link inside the the social links block:

<img width="275" alt="Screenshot 2024-12-11 at 1 04 02 pm" src="https://github.com/user-attachments/assets/36ef0480-1038-4d4b-81c4-72749d457d86">

And in Write mode, the block styles are shown for all links:

<img width="278" alt="Screenshot 2024-12-11 at 1 03 52 pm" src="https://github.com/user-attachments/assets/f210de3d-78dd-4feb-852d-598ff2a13cae">